tests/acceptance/features/apiSharingNg/removeAccessToDriveItem.feature

```diff
@@ -170,48 +170,10 @@ Feature: Remove access to a drive item
       | space           | NewSpace           |
       | permissionsRole | <permissions-role> |
       | password        | %public%           |
-    When user "Alice" removes the share permission of link from file "textfile.txt" of space "NewSpace" using the Graph API
+    When user "Alice" removes the link of file "textfile.txt" from space "NewSpace" using the Graph API
     Then the HTTP status code should be "204"
     Examples:
       | permissions-role |
       | view             |
       | edit             |
       | blocksDownload   |
-
-  
-  Scenario Outline: user removes user member from project space
-    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
-    And user "Alice" has created a space "NewSpace" with the default quota using the Graph API
-    And user "Alice" has sent the following share invitation:
-      | space           | NewSpace           |
-      | sharee          | Brian              |
-      | shareType       | user               |
-      | permissionsRole | <permissions-role> |
-    When user "Alice" removes the share permission of user "Brian" from space "NewSpace" using the Graph API
-    Then the HTTP status code should be "204"
-    And the user "Brian" should not have a space called "NewSpace"
-    Examples:
-      | permissions-role |
-      | Space Viewer     |
-      | Space Editor     |
-      | Manager          |
```

Comment:
We may want to have few coverage for `/drives/{drive-id}/items/{item-id}permissions/{permissionID}` endpoint for space share (unless it is not going to be removed in the future).
Can we do similar to following for root endpoints? https://github.com/owncloud/ocis/blob/4a58fcf5805dbfa5a88a1d5c33673dff63d64045/tests/acceptance/features/apiSharingNg/listPermissions.feature#L922

And then we can also rename existing scenario to:
`When user "Alice" lists the permissions of space "new-space" using permissions endpoint of Graph API`

Reply:
okey, I added this test in the second commit